data/abilities/collection/T1039__data-from-network-shared-drive.yml

```diff
@@ -0,0 +1,12 @@
+- id: ad7029e6-8383-4280-8336-7df4bef53060
+  name: Data from Network Shared Drive
+  description: Ability for Data from Network Shared Drive (ATT&CK T1039). Generated
+    from Atomic Red Team procedures; review and test safely before use.
+  tactic: collection
+  technique:
+    attack_id: T1039
+    name: Data from Network Shared Drive
+  platforms:
+    windows:
+      psh:
+        command: copy \\#{remote}\C$\#{share_file} %TEMP%\#{local_file}
```

Comment:
we typically use the `#[a.b.c}` tuplet format for facts

also, `%TEMP%` should be `$env:temp` in Powershell, since the `%VARNAME%` format is specific to command prompt, not powershell